src/components/CalendarGrid.vue

```diff
@@ -154,6 +168,16 @@ export default {
 			}
 		},
 		eventSources() {
+			if (this.isWidget) {
+				const publicUrl = window.OC.webroot + `/remote.php/dav/public-calendars/${this.token}/`
```

Comment:
I think by using `@nextcloud/router` we should be able to avoid accessing this global variable

Reply:
Actually window.OC.webroot  not needed at all, escaped my testing